charts/opentelemetry-operator/values.yaml

```
@@ -33,6 +37,8 @@ manager:
 kubeRBACProxy:
   image:
     repository: gcr.io/kubebuilder/kube-rbac-proxy
+  ports:
+    proxyPort: 8443
     tag: v0.11.0
```

```suggestion
    tag: v0.11.0
  ports:
    proxyPort: 8443
```

I think this is the cause of the test failures.  The `tag` field needs to be under the `image` section not the `ports` section.  Thank goodness for automated tests.

---

I could've been when I resolved the conflicts - merge human error ?